copier.yml

```diff
@@ -109,10 +109,6 @@ odoo_version:
   type: float
   default: 16.0
   choices:
-    - 7.0
-    - 8.0
-    - 9.0
-    - 10.0
```

Comment:
I don't think this is a good solution. Even if these versions are outdated, you can always get a new customer in one of them and you'd need to scaffold it to migrate.